<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>658</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>521</width>
      <height>411</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>NUMERO 1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_n1">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>40</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_n2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>100</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>NUMERO 2</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_suma">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>SUMA</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_resta">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>RESTA</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_multiplica">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>220</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>MULTIPLICA</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_divide">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>310</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>DIVIDE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_calcular">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>501</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>CALCULAR</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="txttitle">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>511</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
      <strikeout>false</strikeout>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>CALCULADORA</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>521</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>RESULTADO</string>
    </property>
    <widget class="QLabel" name="lbl_resultado">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>491</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
